modules/post/windows/gather/credentials/mremote.rb

```diff
@@ -51,7 +52,7 @@ def get_xml(path)
         condata = read_file(path)
         loot_path = store_loot('mremote.creds', 'text/xml', session, condata, path)
         vprint_good("confCons.xml saved to #{loot_path}")
-        parse_xml(condata)
+        parse_xml(loot_path)
```

Comment:
This could really just be `condata`. There isn't any need to be passing the loot file like this when you have the file data in `condata` already.